### PR TITLE
Add v3.0.1

### PR DESCRIPTION
**Pull request by felipepaes**
Hi everyone o/ 
I'll be helping maintaining the project

@xmnlab ,please, let me know if there is any change needed

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xmnlab** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,59 +27,64 @@ requirements:
     - __unix  # [unix]
     - __win   # [win]
     - backoff >=1.8.0
-    - bleach >=3.0.2,<4.0.0
-    - cachelib >=0.4.1,<0.5
+    - python >=3.8,<=3.11
+    - python
```
Comment (xmnlab) [inline recipe/meta.yaml]:
```suggestion
```

**Comment by xmnlab** [comment]
@felipepaes I asked the conda-forge bot to rerender this feedstock, so please update your branch locally.

additionally, please reset the build number to 0 here: https://github.com/conda-forge/superset-feedstock/pull/29/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR16

as you are proposing a new version, the build number should start with 0

**Comment by xmnlab** [comment]
@felipepaes ... I am not really sure about this error ... but just from the error message, I would suggest to remove these lines https://github.com/conda-forge/superset-feedstock/pull/29/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR27-R28

**Comment by felipepaes** [comment]
@xmnlab I made the changes but it is still failing, maybe I should add these lines back?

```yml
 - __unix  # [unix]
 - __win   # [win]
```

**Comment by SaravananSathyanandhaQC** [comment]
@conda-forge-admin, please rerender

**Comment by SaravananSathyanandhaQC** [comment]
@felipepaes I think the requirements weren't quite right in that version of superset - I would go to 3.0.4 where flask-caching was bumped to 2.x (or just go straight to 3.1.x now)

**Comment by mediocretech** [comment]
closing in favor of newer versions

**Pull request closed without merging**